site/_data/i18n/deprecations_calendar.yml

```
@@ -0,0 +1,35 @@
+filter_by:
+  en: 'Filter by'
+
+name:
+  en: 'Name'
+
+removal_date:
+  en: 'Removal date'
+
+chrome_version:
+  en: 'Chrome version'
+
+article:
+  en: 'Article'
+
+read_article:
+  en: 'Read article'
+
+next_7_days:
+  en: 'Next 7 days'
+
+next_30_days:
+  en: 'Next 30 days'
+
+next_90_days:
+  en: 'Next 90 days'
+
+next_6_months:
+  en: 'Next 6 months'
+
+next_12_months:
+  en: 'Next 12 months'
+
+empty:
+  en: "Sorry, we couldn't find any results that match your selected filters."
```

That's a little to e-commercy I think 🙂 

```suggestion
  en: "There aren't any deprecations that match your selected filters."
```